docs/user-guide/get-started.md

```diff
@@ -122,36 +111,8 @@ You can use the [`overrides`](configure.md#overrides) property. For example, to
 }
 ```
```

Comment:
[suggest] `**/*.{js}` glob pattern is invalid. How about fixing it while we're at it?

https://github.com/stylelint/stylelint/blob/2e49275d0147a702607feba0b1a96993f607551b/docs/user-guide/get-started.md?plain=1#L107

```diff
-     "files": ["**/*.{js}"],
+     "files": ["**/*.js"],
```